views/default/js/languages.php

```diff
@@ -22,6 +22,12 @@
 $all_translations = elgg_get_config('translations');
 $translations = $all_translations['en'];
 
+if ($language != 'en' && !isset($all_translations[$language])) {
+	// try to reload missing translations
```

Comment:
normal elgg only loads a users language and `en`, not all installed translations